bin/admin_es_create_index

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require "datura"
+
+params = Datura::Parser.es_create_delete_index
+options = Datura::Options.new(params).all
+
+put_url = File.join(options["es_path"], "#{options["es_index"]}?pretty=true")
+get_url = File.join(options["es_path"], "_cat", "indices?v&pretty=true")
+
+begin
+  # TODO if we want to add any default settings to the new index,
+  # we can do that with the payloud and then use rest-client again instead of exec
+  # however, rest-client appears to require a payload and won't allow simple "PUT" with none
```

Comment:
It did not accept {} or nil, I didn't actually try empty strings.
How do you require the user to be root?

Reply:
This is how I do it in Bash scripts, with the EUID env var:
```bash
# Check if root
if [[ $EUID -ne 0 ]]; then
  echo "Please run as root"
  exit 1
fi
```

Reply:
I implemented this as `raise "Must run as root user" unless Process.uid == 0` which seems to work fine in irb, but when I added it to the gem it fails out "can't find gem datura with executable admin_es_create_index" which I suspect is possibly RVM / sudo user related.  Since this isn't critical for launch I am going to throw it in an issue instead of trying to work through it now.

Reply:
https://github.com/CDRH/data/issues/132